src/include/dhcp-server/dialogs2.rb

```diff
@@ -1733,11 +1718,9 @@ def HostManagementInit(key)
         Item(
           Id(id2),
           id2,
-          Ops.get(opts, "ip", ""),
-          Ops.get(opts, "hardware", ""),
-          Ops.get(opts, "type", "ethernet") == "ethernet" ?
-            _("Ethernet") :
-            _("Token Ring")
+          opts.fetch("ip", ""),
+          opts.fetch("hardware", ""),
+          _("Ethernet")
```

Comment:
maybe we can remove this column in table?

Reply:
Another place where I was not sure. I'll remove it.